fabric-object-builders-v0/src/main/java/net/fabricmc/fabric/api/villager/v1/VillagerProfessionBuilder.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.villager.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Item;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.util.Identifier;
+import net.minecraft.village.TradeOffers;
+import net.minecraft.village.VillagerProfession;
+import net.minecraft.world.poi.PointOfInterestType;
+
+import net.fabricmc.fabric.impl.object.builder.VillagerProfessionBuilderImpl;
+
+/**
+ * Allows the creation of new {@link VillagerProfession}s.
+ *
+ * <p>Note this does not register any trades to these villagers. To register trades, add a new entry with your profession as the key to {@link TradeOffers#PROFESSION_TO_LEVELED_TRADE}.
+ */
+public interface VillagerProfessionBuilder {
+	/**
+	 * Creates a builder to make a {@link VillagerProfession}.
+	 * @return A new builder.
+	 */
+	static VillagerProfessionBuilder create() {
+		return new VillagerProfessionBuilderImpl();
+	}
+
+	VillagerProfessionBuilder id(Identifier id);
+
+	/**
+	 * The {@link PointOfInterestType} the villager will search for when finding it's workstation.
+	 * @param type The {@link PointOfInterestType}.
+	 * @return this builder.
+	 */
+	VillagerProfessionBuilder workstation(PointOfInterestType type);
+
+	/**
+	 * Items that a Villager may harvest in this profession.
+	 *
+	 * <p>In Vanilla this is used by the farmer to define what type of crops the farmer can harvest.
+	 *
+	 * @param items The items.
+	 * @return this builder.
+	 */
```

Comment:
Small issue, but 
`In Vanilla this` -> `In Vanilla, this`
There's also inconsistent vanilla capitalization in this class-- I think lowercase is more appropriate.

As mentioned above, this param tag doesn't give any useful information. Maybe...
```
@param items  items harvestable by this profession.
```